### DISCUSSION TITLE
Display a node Fx on the Timeline/Xsheet to control keyframes.

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6467] [Ideas]
**Is your feature request related to a problem? Please describe**

The function editor can be a bit tricky for first time users.

**Describe the solution you'd like**

Show the actual Fx node that you are using on Timeline/Xsheet when you click on it to a fast control.

![image](https://user-images.githubusercontent.com/74098734/115610248-7f79e100-a2e0-11eb-934b-a82ba2d9b3ed.png)

An option to get control of all parameters could be a tab like the camera:

![image](https://user-images.githubusercontent.com/74098734/115611675-4b072480-a2e2-11eb-8665-1b4fdcc3e88b.png)

Or simply show the keyframes that you are manipulating in the vertical bar.

**Comment by RodneyBaker** [comment]
It's an interesting idea.
How would that handle multiple FX nodes?

**Comment by F-Burning** [comment]
Being a direct and simple way I would leave it one node at a time, and to do more complex things use the function editor, but, an idea is, when you select multiple, an extensible sub-category could be opened automatically, and closed with a tab at the bottom. The first selected node appears in yellow, and the subcategories in orange (these colors would be optional, of course)

![image](https://user-images.githubusercontent.com/74098734/115617827-cb7d5380-a2e9-11eb-9ef2-b06c05f644eb.png)
